saleor/order/models.py

```diff
@@ -111,10 +111,8 @@ def __str__(self):
         return '#%d' % (self.id, )
 
     def get_total(self):
-        try:
-            return super(Order, self).get_total()
-        except AttributeError:
-            return Price(0, currency=settings.DEFAULT_CURRENCY)
+        # For backwards compatibility
+        return self.total
```

Comment:
To expand on that: we do not have a clear versioning policy which prevents us from having a clear deprecation policy. Until we get one (and we totally should) there is no use in adding deprectation warnings around dead code, let's just remove it and think of what we need to release a "1.0".